src/components/Reports/sharedComponents/ReportPage/ReportPage.css

```diff
@@ -20,4 +19,4 @@
   padding: 24px 24px 0 24px;
   background-color: #f1ebf7;
   border-top-left-radius: 25px;
-}
\ No newline at end of file
+}
```

Comment:
```suggestion
}

@media(max-width:850px){
  .report-page-wrapper {
    display: flex;
    flex-direction: column;
    padding: 40px;
    align-items: center;
  }

  .report-page-profile {
    padding: 24px;
    border-radius: 25px;
  }

  .report-page-content {
    padding: 24px;
  }
}
```

Widths below 850px have the profile component fall off the screen.
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/23086054/93999043-5e2b-4d27-85d6-51e5c2fa0f35)